Sources/PushNotifications.swift

```diff
@@ -211,24 +211,26 @@ import Foundation
      - Precondition: `deviceToken` should not be nil.
      */
     /// - Tag: registerDeviceToken
-    @objc public func registerDeviceToken(_ deviceToken: Data, completion: @escaping () -> Void = {}) {
+    @objc public func registerDeviceToken(_ deviceToken: Data, completion: @escaping (Error?) -> Void = { _ in }) {
         guard
             let instanceId = Instance.getInstanceId(),
             let url = URL(string: "https://\(instanceId).pushnotifications.pusher.com/device_api/v1/instances/\(instanceId)/devices/apns")
         else {
-            print("[Push Notifications] - Something went wrong. Please check your instance id: \(String(describing: Instance.getInstanceId()))")
+            completion(PushNotificationsError.error("[Push Notifications] - Something went wrong. Please check your instance id: \(String(describing: Instance.getInstanceId()))"))
```

Comment:
I'd probably keep the print, just in case someone is integrating it for the first time and didn't use the completion handler.

Reply:
You could always make the completion optional? As long as the completion is called here, I don’t think leaving the print statement would be a problem.